### PR TITLE
Update ks_nordschleife.ini

### PR DESCRIPTION
**Pull request by Linus-Britto**
Fixed Mercedes Logo emissive in GP.

Improved distant glow for all lights.

Added room window lights in GP pits.

Added better winter textures for '3d' trees (only set during winter).

Code optimisation: merged shader_replacement_1 and shader_replacement_2 as both have exact same parameters.

Code optimisation: changed shader_replacement_7 to ksPerPixelAlpha for enabling 3d winter trees during winter.

Code optimisation: removed material_adjustment_31 as it has the same material as material_adjustment_33.

### CODE REVIEW

**Comment by Marocco2** [comment]
Added a comment on your PR

**Comment by Linus-Britto** [comment]
Sorry I didn't get you. I'm new to GitHub.

**Comment by Linus-Britto** [comment]
Well, I thoroughly cheked the entire PR and I cannot see your comments anywhere. On the top right section of the PR it says no reviews as well. Could you be more precise where you've commented (as I don't see a dedicated review section either) or what lines of the code you want to comment on?

File: config/tracks/ks_nordschleife.ini [hunk 1]
```diff
@@ -701,17 +699,11 @@ CONDITION=SEASON_WINTER
...
 [SHADER_REPLACEMENT_7]
-MATERIALS = branch5, Branches_03
-SHADER = $" read('csp/version', 0) >= 2492 and 'ksTree_ppshadows' or 'ksPerPixelAT' "
-BLEND_MODE = ALPHA_TEST
-CAST_SHADOWS = 1
-IS_TRANSPARENT = 0
-DOUBLE_FACE_SHADOW_BIASED = 0
-PROP_... = ksEmissive, -0.001
-PROP_... = ksAlphaRef, 0.2
+MATERIALS = branch?, Branches?, tree_shadow
```
Comment (Linus-Britto) [inline config/tracks/ks_nordschleife.ini:705]: Changed shader to enable 3d winter trees.

**Review by Marocco2 — CHANGES_REQUESTED** [review]
Now it should be visible, my bad. I thought they were sent to github

File: config/tracks/ks_nordschleife.ini [hunk 2]
```diff
@@ -859,7 +861,7 @@ CONDITION=NIGHT_SMOOTH
 
 [Material_DistantEmissive]
 DistantGlowRadius_v2=0.6  ;  most influential
-Meshes=NSLIT_adenauerL0, Torus001, NSLITS_litpostglow?, NSLIT_SingleSpotsGlow?, NSLIT_BigSpots?
+Meshes=NSLIT_adenauerL0, NSLITS_litpostglow?, NSLIT_SingleSpotsGlow?, NSLIT_BigSpots?
```
Comment (Linus-Britto) [inline config/tracks/ks_nordschleife.ini:864]: Removed Mercedes Logo mesh from distant emissive as it causes a bug in emissive rendering.

File: config/tracks/ks_nordschleife.ini [hunk 3]
```diff
@@ -859,7 +861,7 @@ CONDITION=NIGHT_SMOOTH
 
 [Material_DistantEmissive]
 DistantGlowRadius_v2=0.6  ;  most influential
-Meshes=NSLIT_adenauerL0, Torus001, NSLITS_litpostglow?, NSLIT_SingleSpotsGlow?, NSLIT_BigSpots?
```
Comment (Linus-Britto) [inline config/tracks/ks_nordschleife.ini:862]: Removed Mercedes Logo mesh from distant emissive as it causes a bug in emissive rendering.

File: config/tracks/ks_nordschleife.ini [hunk 4]
```diff
@@ -1221,15 +1328,6 @@ KEY_0=ksEmissive
...
 
-[MATERIAL_ADJUSTMENT_31]
-ACTIVE=1
-VISIBILITY_LEVEL=0
-DESCRIPTION=Dorint Hotel windows
-MATERIALS=MATdorintwin?
-KEY_0=ksEmissive
-VALUE_0=255,255,255,0.006
-VALUE_0_OFF=0
-
```
Comment (Linus-Britto) [inline config/tracks/ks_nordschleife.ini:1232]: Removed material_adjustment_31 as it is redundant with material_adjustment_33 and serves no purpose.

File: config/tracks/ks_nordschleife.ini [hunk 5]
```diff
@@ -540,7 +538,7 @@ CONDITION=SEASON_SUMMER
 ACTIVE=1
 VISIBILITY_LEVEL=0
 DESCRIPTION=summer settings for road + curbs + walls + stuff
-MATERIALS=concrete, park-dirt, tavr, armco, pitlane-multimap?, Vehicles, box, asphalt-, pitlane-multilayer, parkAO-shad, t13, karussell, Kleinekarussell, concgrid, brickfloor?, wall1, curb-new, tyres, asph shad, asph shad dark, pavement-brick
```
Comment (Linus-Britto) [inline config/tracks/ks_nordschleife.ini:543]: Removed box mesh from summer settings as it renders GP cylindrical windows in a milky shade during summer with CSP windows.

**Pull request merged**